content_scripts/CameraVis.js

```diff
@@ -359,15 +360,32 @@ void main() {
                 key,
                 container: this.container.matrix.elements,
                 phone: this.phone.matrix.elements,
-                texture: this.texture.image.toDataURL(),
+                texture: this.texture.image.toDataURL('image/jpeg', 0.7),
```

Comment:
It would be nice if the key has some 'id' of which camera it comes from rather than a random number.

Reply:
Added camera's id to patch serialization format